src/engraving/rw/compat/read302.h

```diff
@@ -39,6 +40,8 @@ class Read302
 
     static Err read302(MasterScore* masterScore, XmlReader& e, ReadContext& ctx);
 
+    static void fixInstrumentId(Instrument* instrument);
```

Comment:
This is a very strange place for this function. I would suggest making it private for Instrument and calling it from Instrument::updateInstrumentId()

Reply:
I think it's not _so_ strange, since this function is purely related to reading of 3.6 scores. I would not like to "clutter" the Instrument class with auxiliary single-use methods like this.
However, while thinking about your comment, I realized that this applies to 3.6.x scores _only_, and not to older scores (because for older scores, there were no instrument ids at all). So I'll change that. That hopefully improves the situation :)